forum/projects/tests.py

```diff
@@ -1,3 +1,26 @@
-from django.test import TestCase
+from rest_framework.reverse import reverse
+from rest_framework import status
+from forum.tests_setup import TestUserSetupMixin
 
-# Create your tests here.
+from projects.models import Industry
+
+
+class IndustryModelTestCase(TestUserSetupMixin):
+    CREATE_INDUSTRY_COUNT: int = 3
+
+    def setUp(self) -> None:
+        self.setup_test_user()
+
+        # create industries
+        for i in range(self.CREATE_INDUSTRY_COUNT):
+            tmp_industry = Industry.objects.create(
+                name=f"industry {i}",
+                description=f"test description {i}"
+            )
+            Industry.objects.get(industry_id=tmp_industry.industry_id)
```

Comment:
The retrieval of `Industry` objects immediately after creation in the loop seems unnecessary